x/ibc/applications/transfer/keeper/relay.go

```diff
@@ -223,9 +272,30 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 	}
 
 	// send to receiver
-	return k.bankKeeper.SendCoinsFromModuleToAccount(
+	if err := k.bankKeeper.SendCoinsFromModuleToAccount(
 		ctx, types.ModuleName, receiver, sdk.NewCoins(voucher),
-	)
+	); err != nil {
+		return err
+	}
+
+	defer func() {
+		telemetry.SetGaugeWithLabels(
+			[]string{"ibc", types.ModuleName, "packet", "receive"},
+			float32(data.Amount),
+			[]metrics.Label{telemetry.NewLabel("denom", data.Denom)},
+		)
+
+		telemetry.IncrCounterWithLabels(
```

Comment:
ditto

Reply:
ditto with the chain ID or that the counter is not useful?

Reply:
added chain id from counterparty